.github/workflows/release.yml

```diff
@@ -58,6 +58,26 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
 
+  deploy-dev:
+    runs-on: ubuntu-latest
+    needs: dockerize
+    environment: dev
```

Comment:
Should this be `prod`?

Reply:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/14722250/193414863-ad2cf10e-205c-494f-b18b-9a2ab39d3cc9.png">